### PR TITLE
rakudo-star: update to 2018.10

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rakudo-star.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class RakudoStar < Formula
   desc "Perl 6 compiler"
   homepage "https://rakudo.org/"
-  url "https://rakudo.perl6.org/downloads/star/rakudo-star-2018.06.tar.gz"
-  sha256 "309fbaaf441866ee9454451e83e90e4a21391944d475eacda93f48c7671da888"
+  url "https://rakudo.perl6.org/downloads/star/rakudo-star-2018.10.tar.gz"
+  sha256 "a4914220b58de6b3298adfc80d740a1fb69dbb1fed6155e563eb0a9d4b63a346"
   revision 1
```
Comment (fxcoudert) [inline Formula/rakudo-star.rb]: `'revision 1' should be removed`

Comment (yurikoles) [reply Formula/rakudo-star.rb]: done

**Comment by lembacon** [comment]
The build on Sierra has been stuck for 2 days and 23 hours, killed and re-queued.

**Comment by scpeters** [comment]
I just killed the sierra build again after hanging for 13 hours.

**Comment by yurikoles** [comment]
Is there any possibility to debug this? Or simply depend on higher macos version?

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by scpeters** [comment]
I'm testing this build on a sierra machine right now, and it seems to be hanging during `make install`. I've posted the console output of `brew install rakudo-star --verbose --debug` here:

* https://gist.github.com/scpeters/484ec4a5548534b4c7df88bd8bc3ddd5

**Comment by yurikoles** [comment]
@scpeters thanks!

P.S. Could you please explain to me how to push to foreign PR?

**Review by fxcoudert — APPROVED** [review]

**Comment by scpeters** [comment]
@yurikoles I was able to push because of the "Allow edits from maintainers" setting

* https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Pull request closed without merging**